challengeA/ishiwata/lib/energycharge.rb

```diff
@@ -0,0 +1,49 @@
+
+class EnergyCharge
+  attr_reader :usage
+
+  PLAN = ["従量電灯B", "おうちプラン", "ずっとも電気1"]
+  CHARGE = ["tokyo_energy", "looop_energy", "tokyogas_energy"]
+
+  def initialize(usage)
+    @usage = usage
+  end
+
+  def choose_plan(plan_name)
+    plan = PLAN.index(plan_name)
+    self.send(CHARGE[plan])
+  end
+
+
+  def tokyo_energy
+    energy_charge =
+      if usage > 300
+        30.57 * (usage - 300) + 26.48 * 180 + 19.88 * 120
```

Comment:
このあたりのマジックナンバーも CSV 等の形式で
別に料金マスターを作るとより変更に強い設計にできる。

Reply:
`180` という数字はベタに書かず 
```suggestion
        30.57 * (usage - 300) + 26.48 * (300 - 120) + 19.88 * 120
```
などとする方が後から読む人にとって、 書いたときの意図を読み取りやすくなる。